Error: Cannot find module './package.json' when running npx @expressjs/codemod upgrade

When I run npx `@expressjs/codemod upgrade`, I encounter the following error:

```bash
✘ npx @expressjs/codemod
node:internal/modules/cjs/loader:1247
  throw err;
  ^

Error: Cannot find module './package.json'
Require stack:
- /home/tanya/.npm/_npx/2fe7c9c2aa51edce/node_modules/@expressjs/codemod/build/index.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1244:15)
    at Function._load (node:internal/modules/cjs/loader:1070:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
    at Module.require (node:internal/modules/cjs/loader:1335:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (/home/tanya/.npm/_npx/2fe7c9c2aa51edce/node_modules/@expressjs/codemod/build/index.js:12:40)
    at Module._compile (node:internal/modules/cjs/loader:1562:14)
    at Object..js (node:internal/modules/cjs/loader:1699:10)
    at Module.load (node:internal/modules/cjs/loader:1313:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/tanya/.npm/_npx/2fe7c9c2aa51edce/node_modules/@expressjs/codemod/build/index.js'
  ]
}

Node.js v22.13.1
```

## Environment information

Platform:
`Linux pc-685236 5.15.167.4-microsoft-standard-WSL2 #1 SMP Tue Nov 5 00:21:55 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux`

Node.js version:
`Node.js v22.13.1`

### Any other relevant information: The `package.json` file is present in the module directory.

## What steps will reproduce the bug?

* Run npx @expressjs/codemod upgrade
* Observe the error message

## Investigation
I investigated the issue in build/index.js and noticed that the package.json file is located at the root of the codemod module. However, in index.js, it is assumed that package.json is in the build/ directory. To fix this, I changed the path from ./package.json to ../package.json, and the issue was resolved. Here is the modified line of code:
```js
const package_json_1 = __importDefault(require("../package.json"));
```

--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
     "url": "git+https://github.com/expressjs/codemod.git"
   },
   "keywords": ["codemods", "express"],
-  "files": ["build/**/*.js"],
+  "files": ["build"],
   "scripts": {
     "clean": "rm -rf build",
     "dev": "tsc -d -w -p tsconfig.json",

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "module": "commonjs",
-    "sourceMap": true,
+    "sourceMap": false,
     "esModuleInterop": true,
     "target": "ES2022",
     "downlevelIteration": true,

